pulpito/templates/job.html

```diff
@@ -77,6 +77,14 @@ <h1 id="tables"><a href="/{{ job.name }}/">{{ job.name }}</a><span class='spaced
                     </a>
                 </p>
               {% endif %}
+              {% if job.pcp_grafana_url %}
+                <p>
+                    PCP+Grafana:
```

Comment:
Perhaps better to make the string displayed to users a little more rube-friendly (like, "Perf Graphs" or something)?